.devops/code-review-pipelines.yml

```
@@ -12,7 +13,7 @@ resources:
     - repository: pagopaEcommerceLocal
       type: github
       name: pagopa/pagopa-ecommerce-local
-      ref: main
+      ref: CHK-4331-ecommerce-cdc-tests
```

```suggestion
      main
```
TODO change to main after ecommerce-local PR merge